### PR TITLE
PHP: implement empty_stream interop test

### LINKED ISSUE #3608: implement interop test empty_stream in PHP
I get the message `Unsupported test case empty_stream`.

### PR DESCRIPTION
**Pull request by stanley-cheung**
Fixes #3608

### CODE REVIEW

File: src/php/tests/interop/interop_client.php [hunk 1]
```diff
@@ -251,6 +251,22 @@ function pingPong($stub) {
...
+ * Passes when run against the Node server as of 2015-10-09
+ * @param $stub Stub object that has service methods.
+ */
+function emptyStream($stub) {
+  // for the current PHP implementation, $call->read() will wait
+  // forever for a server response if the server is not sending any.
+  // so this test is imeplemented as a timeout to indicate the absence
+  // of receiving any response from the server
+  $call = $stub->FullDuplexCall(array('timeout' => 100000));
+  hardAssert($call->read() === null, 'Server returned too many responses');
```
Comment (murgatroid99) [inline src/php/tests/interop/interop_client.php]: Don't you have to call `writesDone` or something?

Comment (stanley-cheung) [inline src/php/tests/interop/interop_client.php]: Right, thanks for pointing it out. I also discovered that without calling `writesDone` the internal call will be messed up if I invoke the test a second time.

**Comment by stanley-cheung** [comment]
Jenkins is mostly green

**Comment by jtattermusch** [comment]
Thanks!

**Pull request merged**